lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -679,6 +679,23 @@ function dosomething_reportback_preprocess_entity(&$variables) {
     $variables['theme_hook_suggestions'][] = 'reportback';
     $entity = $variables['elements']['#entity'];
   }
+
+  if (variable_get('rogue_collection', FALSE)) {
+    $rogue_base_url = variable_get('dosomething_rogue_url', 'https://rogue.dosomething.org/api');
+    $apiUrl = $rogue_base_url . '/v1/reportbacks';
```

Comment:
ah brilliant, thank you! It seems like when I click more, the reportbacks are being populated but doesn't seem like from Rogue. I don't know what I'm missing though?